tests/lib/fakestore/store/store.go

```diff
@@ -476,6 +478,153 @@ func (s *Store) collectAssertions() (asserts.Backstore, error) {
 	return bs, nil
 }
 
+type contextSnap struct {
+	SnapID      string `json:"snap-id"`
+	InstanceKey string `json:"instance-key"`
+}
+
+type installRefreshAction struct {
+	Action      string `json:"action"`
+	InstanceKey string `json:"instance-key"`
+	SnapID      string `json:"snap-id"`
+	Name        string `json:"name"`
+}
+
+type installRefreshReqJSON struct {
```

Comment:
Everything here is "JSON".. is that a Request to the store while the one below is the Result? If so just Request might do. Similar comments for the types below.